saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/property/PairwiseConjunction.scala

```diff
@@ -15,8 +15,8 @@ object PairwiseConjunction {
   def concat[T](a: Property[T], b: List[Property[T]], y: T): List[String] =
```

Comment:
Could you add javadoc to these two functions? (and potentially rename their variables to sth meaningful?)